src/jsutils/didYouMean.js

```diff
@@ -19,12 +19,16 @@ export default function didYouMean(firstArg, secondArg) {
       ? [firstArg, secondArg]
       : [undefined, firstArg];
 
+  const suggestionsArgFromSet = Array.from(new Set(suggestionsArg)).filter(
+    x => x,
+  );
+  console.log({ suggestionsArgFromSet });
```

Comment:
It's an internal function that has Flow typings to accept only strings so this check is done by Flow linter.
Also, please review your PRs before submission and try to remove stuff like `console.log`.

Reply:
Pardon